examples/TeamInfo.ts

```diff
@@ -0,0 +1,18 @@
+import * as HelloSignSDK
+  from "@hellosign/openapi-javascript-sdk";
+
+const api = new HelloSignSDK.TeamApi();
+
+// Configure HTTP basic authorization: api_key
+api.username = "YOUR_API_KEY";
+
+// or, configure Bearer (JWT) authorization: oauth2
+// $config->setAccessToken("YOUR_ACCESS_TOKEN");
+
+const result = api.teamInfo("team_id");
```

Comment:
Was this sample generated by the typescript SDK? 
I'd expect this team id parameter to be broken out into it's own type. Something like this: 
```json
const data: HelloSignSDK.TeamGetInfoRequest = {
  team_id: "21651sdf5161",
};
```